pkgs/applications/networking/p2p/lokinet/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, libevent
+, libsodium
+, libuv
+, pkg-config
+, sqlite
+, systemd
+, unbound
+, zeromq
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lokinet";
+  version = "0.9.9";
+
+  src = fetchFromGitHub {
+    owner = "oxen-io";
+    repo = "lokinet";
+    rev = "v${version}";
+    sha256 = "AaGsRg9S9Cng9emI/mN09QSOIRbE+x3916clWAwLnRs=";
+    fetchSubmodules = true;
```

Comment:
Can we at least use our nlohmann and pybind11?

Reply:
I don't think so, because I still get this line when using our nlohmann :
```
-- Using the single-header code from /build/source/external/nlohmann/single_include/
```
I'm not sure about pybind11.

Reply:
We could link our version of the library to where the package is expecting it. Especially for security related libraries like openssl or curl this should be done.